src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo"
+          alt="moyo logo"
+          src="images/logo.png"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="index.html"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link diff-color"
+              data-qa="hover"
+              href="index.html"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="index.html"
```

Comment:
The `href` property of the `a` tag should not contain spaces. Anchor links should start with the `#` symbol. If you want to make a link not leading anywhere you can use `href="#"`.